services/consensusalgo/leanhelixconsensus/test/harness.go

```diff
@@ -37,20 +37,36 @@ type harness struct {
 }
 
 func newLeanHelixServiceHarness(auditBlocksYoungerThan time.Duration) *harness {
-	gossip := &gossiptopics.MockLeanHelix{}
-	gossip.When("RegisterLeanHelixHandler", mock.Any).Return().Times(1)
+	h := &harness{
+		gossip:                 &gossiptopics.MockLeanHelix{},
+		blockStorage:           &services.MockBlockStorage{},
+		consensusContext:       &services.MockConsensusContext{},
+		auditBlocksYoungerThan: auditBlocksYoungerThan,
+	}
 
-	blockStorage := &services.MockBlockStorage{}
-	blockStorage.When("RegisterConsensusBlocksHandler", mock.Any).Return().Times(1)
+	h.resetMocks()
```

Comment:
why reset mocks at the start up the test?

Reply:
ok, the term reset is not really good here

Reply:
Reset in the mock object means "clear all setup". but in the harness it means - switch back to harness defaults. so how about "ResetToDefault" ?...

Reply:
`resetAndApplyMockDefaults`?

Reply:
ja